example/profiles/example.profile

```diff
@@ -24,3 +24,4 @@ selections:
     - sshd_disable_empty_passwords
     - sshd_idle_timeout_value=5_minutes
     - sshd_set_idle_timeout
+    - sshd_set_keepalive
```

Comment:
Can you add a `requires` to sshd_set_keepalive and sshd_set_idle_timeout?

Reply:
`sshd_set_idle_timeout` already has it. I'll add to `sshd_set_keepalive` as well.